src/server/game/Battlegrounds/ArenaTeam.cpp

```diff
@@ -315,25 +315,30 @@ void ArenaTeam::SetCaptain(ObjectGuid guid)
 
 void ArenaTeam::DelMember(ObjectGuid guid, bool cleanDb)
 {
+    Player* player = ObjectAccessor::FindPlayer(guid);
```

Comment:
@jackpoz, I would not know if it would be correct FindConnectedPlayer

Reply:
@jackpoz can you review it please?

Reply:
@jackpoz: Possibly the player is removed by the leader, while the player loads screen.